core-common/src/main/java/org/apache/kylin/common/KylinConfigBase.java

```diff
@@ -1457,6 +1457,10 @@ public int getQueryMaxCacheStatementInstancePerKey() {
         return Integer.parseInt(this.getOptional("kylin.query.statement-cache-max-num-per-key", String.valueOf(50)));
     }
 
+    public boolean isQueryPreparedStatementCacheEnable() {
+        return Boolean.parseBoolean(this.getOptional("kylin.query.statement-cache-enabled", "false"));
```

Comment:
Gang, can we enable it by default? It is a good feature.

Reply:
Sure